openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -108,6 +111,32 @@ def student_view(self, context=None):
         """
         fragment = Fragment()
 
+        # Head dependencies
+
+        discussion_vendor_js = ['js/jquery.autocomplete.js', 'js/src/tooltip_manager.js']
```

Comment:
@itsjeyd nit: I would suggest extracting this into a method here or in dedicated module; when we get to course discussions (aka DiscussionCourseXBlock) we'll likely need to do something quite similar.